docs/apis/subsystems/availability/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Availability API
+tags:
+  - Availability
+  - core_availability
+---
+
+The availability API controls access to activities and sections. For example, a teacher could restrict access so that an activity cannot be accessed until a certain date, or so that a section cannot be accessed unless users have a certain grade in a quiz.
+
+The conditional availability system defaults to off; users enable it from the advanced features page in settings. You can still call the API functions even if the system is turned off.
```

Comment:
It has changed and currently the "Enable restricted access" advanced feature is enabled by default.

Reply:
Awesome. I love it when people update the docs when they make such changes 😆

Reply:
For the record, this has been enabled since 3.1: https://tracker.moodle.org/browse/MDL-52219